include/cutlass/arch/arch.h

```diff
@@ -57,7 +57,7 @@ int LaneId() {
 CUTLASS_DEVICE
 int SmId() {
   int ret;
-#if defined(ENABLE_NVPTX)
+#if !defined(CUTLASS_ENABLE_SYCL) || defined(__SYCL_CUDA_ARCH__)
```

Comment:
I did not understand this, when we DO NOT have sycl enabled, it moved the value of laneid register in ret, what about when SYCL is enabled ?

Reply:
This means if sycl is not defined and Nvidia backend is only used or when sycl is defined only for Nvidia backend

Reply:
Aha, I misread the brackets,  that clears it up